insights/mappers/netstat.py

```diff
@@ -285,22 +307,40 @@ def add_data(self, line):
             self.data[i - 1].append(line[from_index: indexes[i]].strip())
             from_index = indexes[i]
             i += 1
-
         self.data[i - 1].append(line[indexes[i - 1]:])
+
         self.datalist.append({m: d for m, d in zip(
             NETSTAT_SECTION_ID[self.name], [r[-1] for r in self.data]
         )})
+        # For convenience, unpack 'PID/Program name' into 'PID' and 'Program name'
+        # This field must exist because of NETSTAT_SECTION_ID and the
+        # exception in add_meta_data
+        pidprogram = self.datalist[-1]['PID/Program name']
+        if '/' in pidprogram:
+            pid, program = pidprogram.split('/')
+            self.datalist[-1]['PID'] = pid
+            self.datalist[-1]['Program name'] = program
+        # For convenience, unpack 'Local Address' into 'Local IP' and 'Port'
+        if 'Local Address' in self.datalist[-1]:
+            local_addr = self.datalist[-1]['Local Address']
+            # Assert rather than throw exception here because Netstat's format
+            # isn't going to change, since it's deprecated.
+            assert ':' in local_addr
```

Comment:
@PaulWay can you explain this a bit further? Exceptions should be used instead of asserts in production.